CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Added
+
+- Add a NOTES column to the output of listing and getting App resources. The column contains information why the last Helm release attempt failed if so, empty otherwiseg
```

Comment:
```suggestion
- Add a NOTES column to the output of listing and getting App resources. The column contains information why the last Helm release attempt failed if so, empty otherwise.
```